cmd/wallet-web/src/store/modules/options.js

```diff
@@ -129,163 +129,61 @@ export default {
             },
           },
           config: {
-            storageType: defaultAgentStartupOpts.storageType,
-            kmsType: defaultAgentStartupOpts.kmsType,
+            storageType: defaultAgentStartupOpts['storage-type'],
+            kmsType: defaultAgentStartupOpts['kms-type'],
             localKMSPassphrase: defaultAgentStartupOpts.localKMSPassphrase,
           },
         });
 
         readCredentialManifests = readManifests();
       }
 
+      let optValue = function (opt) {
+        return opt in agentOpts ? agentOpts[opt] : defaultAgentStartupOpts[opt];
+      };
+
```

Comment:
This might be a bit cleaner:
```suggestion
      const optValue = (opt) => opt in agentOpts ? agentOpts[opt] : defaultAgentStartupOpts[opt];
```